AndroidNativeLoginTemplate/app/src/main/java/com/salesforce/androidnativelogintemplate/NativeLogin.kt

```diff
@@ -0,0 +1,327 @@
+    handleWebviewFallbackResult: ActivityResultLauncher<Intent>? = null,
+    webviewLoginIntent: Intent? = null,
+    shouldShowBack: Boolean = false,
+    backAction: () -> Unit,
+) {
+    LoginTheme {
+        Scaffold(
+            topBar = {
+                Row(modifier = Modifier.statusBarsPadding()) {
+
+                    // Back button should only be shown if there is a user already logged in.  But not
+                    // in the case of Biometric Authentication.
+                    if (shouldShowBack) {
+                        TextButton(onClick = { backAction() }) {
+                            Image(
+                                painter = painterResource(id = com.salesforce.androidsdk.R.drawable.sf__action_back),
+                                colorFilter = ColorFilter.tint(MaterialTheme.colorScheme.primary),
+                                contentDescription = "Back",
+                                modifier = Modifier.padding(start = 16.dp)
+                            )
+                        }
+                    }
+                }
+            },
+            bottomBar = {
+                Column(
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    modifier = Modifier
+                        .navigationBarsPadding()
+                        .fillMaxWidth(),
+                ) {
+                    // Fallback to web based authentication.
+                    TextButton(onClick = { handleWebviewFallbackResult?.launch(webviewLoginIntent) }) {
+                        Text(text = "Looking for Salesforce Log In?")
+                    }
+                }
+            },
+        ) { innerPadding ->
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier
+                    .verticalScroll(rememberScrollState(), reverseScrolling = true)
+                    .navigationBarsPadding()
+                    .padding(top = innerPadding.calculateTopPadding())
+                    .imePadding()
+                    .fillMaxSize(),
+            ) {
+                Spacer(modifier = Modifier.height(75.dp))
+                ElevatedCard(
+                    modifier = Modifier
+                        .padding(16.dp)
+                        .navigationBarsPadding()
+                        .widthIn(350.dp, 500.dp),
+                    elevation = CardDefaults.cardElevation(defaultElevation = 6.dp)
+                ) {
+                    Spacer(modifier = Modifier.height(50.dp))
+                    Image(
+                        painter = painterResource(id = R.drawable.sf__salesforce_logo),
+                        colorFilter = ColorFilter.tint(MaterialTheme.colorScheme.primary),
+                        contentDescription = "",
+                        modifier = Modifier.align(Alignment.CenterHorizontally),
+                    )
+                    UserNamePasswordInput(login)
```

Comment:
@JohnsonEricAtSalesforce I made this a separate composable because I thought it would make it easier to if else `UserNamePasswordInput` and an `OTPInput` composable.  But you are free to handle it however you see fit 😄